test/sequential/sequential.status

```diff
@@ -22,4 +22,5 @@ test-benchmark-child-process  : PASS,FLAKY
 [$jsEngine==chakracore]
 
 [$jsEngine==chakracore && $system==linux]
-test-child-process-pass-fd : PASS,FLAKY
+# This test crashes Jenkins in Linux CI
+test-child-process-pass-fd : SKIP
```

Comment:
if this test crashes Jenkins in Linux CI, why is it under `$jsEngine==chakracore` condition? Or is it only crashing CI setup for chakracore?